app/config/config.neon

```diff
@@ -445,34 +418,52 @@ services:
   - App\Model\View\PipelineViewFactory
   - App\Model\View\InstanceViewFactory
 
-doctrine:
-  user: 'root'
-  password: ''
-  host: localhost
-  dbname: 'recodex-api'
-  autoGenerateProxyClasses: 2 # only generate proxies if their files do not exist
-  charset: utf8mb4
-  defaultTableOptions:
+nettrine.dbal:
+  debug:
+    panel: %debugMode%
+    sourcePaths: [%appDir%]
+  connection:
+    driver: pdo_mysql
+    host: localhost
+    dbname: 'recodex-api'
+    user: 'root'
+    password: ''
     charset: utf8mb4
-    collate: utf8mb4_unicode_ci
-  metadata:
-    App\Model\Entity: %appDir%/model/entity
-  types:
-    bit: Doctrine\DBAL\Types\BooleanType
-    datetime: DoctrineExtensions\DBAL\Types\UTCDateTimeType
-  dql:
-    string:
+    types:
+      bit: Doctrine\DBAL\Types\BooleanType
+      datetime: DoctrineExtensions\DBAL\Types\UTCDateTimeType
+    defaultTableOptions:
+      charset: utf8mb4
+      collate: utf8mb4_unicode_ci
+
+nettrine.orm:
+  configuration:
+    autoGenerateProxyClasses: true
+    customStringFunctions:
       coalesce_sub: DoctrineExtensions\Query\Functions\CoalesceSubselectsFunction
       type: DoctrineExtensions\Query\Functions\TypeFunction
 
-fixtures:
-  locale: "en_US"
-  seed: 1
+nettrine.orm.annotations:
+  mapping:
+   App\Model\Entity: %appDir%/model/entity
 
-annotations:
+nettrine.annotations:
   ignore:
     - LoggedIn
     - POST
     - GET
     - PUT
     - DELETE
+
+nettrine.extensions.atlantic18:
+    softDeleteable: true
```

Comment:
Is this the same as soft-deletable plugin we have been using, or did your replace it alltogethrer?

Reply:
We used `gedmo/doctrine-extensions` https://github.com/Atlantic18/DoctrineExtensions  
Aand, we are still using it through `nettrine/extensions-atlantic18` which is nette integration  
Soft-deletable functionality/filter is part of the doctrine-extensions.

Reply:
I.C.